<launch> 

<!-- ##### Parameters ##### -->

<!-- command="$(find xacro)/xacro '$(find ur_description)/urdf/ur5_joint_limited_robot.urdf.xacro'" /> -->
       <!-- command="$(find xacro)/xacro \-\-inorder '$(find ur_description)/urdf/ur5_robot.urdf.xacro'" /> -->


<!-- Robot Setup -->
<param name="/robot_description" 
       command="$(find xacro)/xacro '$(find ur_description)/urdf/ur5_joint_limited_robot.urdf.xacro'" />
<!-- <param name="/robot_description" 
       textfile="$(find ur_fk_ik)/resource/ur5.urdf" /> -->
<!-- xacro: in-order processing became default in ROS Melodic. You can drop the option. -->
<param name="base_link" type="str" value="base_link"    />
<param name="end_link"  type="str" value="wrist_3_link" />

<!-- Node Setup -->
<param name="UR_FKservice_TOPIC" type="str" value="UR_FK_IK/FKsrv" />
<param name="UR_IKservice_TOPIC" type="str" value="UR_FK_IK/IKsrv" />

<!-- Stage 1 -->
<param name="IK_timeout"  type="double" value="0.0850" />
<param name="IK_epsilon"  type="double" value="0.1200" /> 
<param name="IK_fuzz"     type="double" value="0.1745" />
<param name="IK_samples"  type="int"    value="1"      />

<!-- Stage 2 -->
<param name="IK_timeout2" type="double" value="0.0400" />
<param name="IK_epsilon2" type="double" value="0.0005" /> 
<param name="IK_fuzz2"    type="double" value="0.0873" />
<param name="IK_samples2" type="int"    value="5"      />

<!-- ##### Service Node ##### -->

<node pkg="ur_fk_ik" name="UR_FK_IK" type="FK_IK" output="screen" />

<!-- <node name="talker" pkg="rospy_tutorials" type="talker" /> -->


</launch>